.travis.yml

```diff
@@ -3,17 +3,21 @@ language: node_js
 
 # Support Active LTS versions of Node.js
 node_js:
-  - "10"
   - "12"
+  - "14"
+
+branches:
+  only:
+    - master
+    - develop
+    - /^v(\d+\.)?(\d+\.)?(\*|\d+)$/
```

Comment:
I guess we could remove it

Reply:
So with the fact that we want on tags to run the deploy script and we want any PR to master & develop to be run on travis (but without deploy). 
Can you explain why we could remove it ? To my understanding it look it might forbid the initial run of tests on a tag push ?

Reply:
I meant only tag regexp coz it doesn't work for `branches.only` anyways.

Reply:
Hum. I'm not sure myself, in doubt, I remove it following you call, worst situation is it needing to be re-added.

Reply:
Yeah, sure.

FYI https://docs.travis-ci.com/user/customizing-the-build#building-specific-branches